.github/workflows/build-java.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Java
+
+on: workflow_dispatch
+
+permissions:
+  contents: read
+
+jobs:
```

Comment:
Will these automatically run on every build?  We had a security issue where an attacker modified the build in a PR with a script that could have gained access to secrets leveraged in the build.  Since then we've put all automated builds behind a gate in screwdriver that requires a committer to approve the build.

Reply:
No these are setup to only run if [manually triggered](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow).

`on: workflow_dispatch`